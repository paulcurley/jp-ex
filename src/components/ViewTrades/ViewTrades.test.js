import React from 'react';
import { shallow } from 'enzyme';
import ReactDOM from 'react-dom';
import { ViewTrades } from '.';

const setup = (setupProps = {}) => {
    const props = {
        ...setupProps,
    };

    const wrapper = shallow(<ViewTrades {...props} />);

    return {
        props,
        wrapper,
    };
};
describe('View Trades', () => {
    it('render', () => {
        const wrapper = setup({
            trades: [
                {
                    stockSymbol: 'ALC',
                    price: 432,
                    noOfShares: 54,
                },
            ],
        });
        expect(wrapper).toMatchSnapshot();
    });
});
