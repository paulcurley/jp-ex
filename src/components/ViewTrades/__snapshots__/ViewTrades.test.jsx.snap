// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`View Trades render 1`] = `
Object {
  "props": Object {
    "trades": Array [
      Object {
        "noOfShares": 54,
        "price": 432,
        "stockSymbol": "ALC",
      },
    ],
  },
  "wrapper": <table>
    <thead>
      <tr>
        <th>
          Stock Symbol
        </th>
        <th>
          Price
        </th>
        <th>
          Number of Shares 
        </th>
      </tr>
    </thead>
    <tbody>
      <tr
        key="0"
      >
        <td>
          ALC
        </td>
        <td>
          432
        </td>
        <td>
          54
        </td>
      </tr>
    </tbody>
  </table>,
}
`;
